block_to_imp.c

```diff
@@ -183,7 +189,13 @@ static struct trampoline_set *alloc_trampolines(char *start, char *end)
 		memcpy(block, start, end-start);
 	}
 	metadata->buffers->headers[HEADERS_PER_PAGE-1].block = NULL;
+
+	#if defined(__powerpc64__) // Workaround as pagesize is 64k on ppc64
```

Comment:
It would be nice to have the page size in a header so that we can ensure consistency with what the assembly expects.

Reply:
Are you ok with it to just hard code the page size?

Reply:
I think I will address this in a different PR later.

Reply:
I’m happy to hard-code the page size, we would need to dynamically patch the assembly if we changed page sizes and that’s too hard.